test/globalize/fallbacks_test.rb

```diff
@@ -93,6 +93,20 @@ class FallbacksTest < MiniTest::Spec
       assert_equal 'foo', post.title
     end
 
+    it 'resolves fallbacks with empty translations' do
+      Globalize.fallbacks_for_empty_translations = true
+      I18n.fallbacks.map :'de-DE' => [ :'en-US' ]
+      post = Post.create :title => 'foo'
```

Comment:
The only reason I used the old hashrocket syntax is because the other tests use it as well. Just trying to keep the code base consistent :+1: